Tests/MapperTests/RawRepresentibleValueTests.swift

```diff
@@ -130,6 +130,38 @@ final class RawRepresentibleValueTests: XCTestCase {
         XCTAssertNil(test.value)
     }
 
+    func testConvertibleArrayOfKeysDoesNotThrow() {
+        struct Test: Mappable {
+            let value: Value?
+            init(map: Mapper) throws {
+                try self.value = map.from(["a", "b"])
+            }
+        }
+
+        enum Value: String {
+            case first = "hi"
```

Comment:
I know I haven't been doing this everywhere yet but you can remove the explicit string here and just use `"first"` in the test

Reply:
Makes sense, pushed another commit which updates all the cases in this file to use implicit raw values